catalog/acm/acm-membership-api.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Enable the Container service in the platform project
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
```

Comment:
Lets call this file `services.yaml` to be consistent with other blueprints

Reply:
This would also be a good opportunity to use https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/master/functions/go/enable-gcp-services to describe this succinctly

Reply:
Renamed the file for now. I will try to use the function for the next iteration.